jd-geek-source/src/app/components/test-diplay/test-diplay.component.html

```diff
@@ -0,0 +1,18 @@
+<div>
+  <h4>Testing Getting Data From The Best Buy API</h4>
+
+  <div *ngFor="let product of testResults">
+    <div>
+      <p>
```

Comment:
`p` tag should only be used for paragraph text for better clarity, both in code and on the dom. There are other added benefits in terms of managing your view, as well, as it to pertains to consist styling across the site/app.